Community/ideAlarm/automation/lib/python/community/idealarm/__init__.py

```diff
@@ -0,0 +1,504 @@
+# -*- coding: utf-8 -*-
+
+import weakref # Using this to prevent problems with garbage collection
+
+from core.jsr223 import scope
+from core.log import logging, LOG_PREFIX
+from core.utils import isActive, getItemValue, postUpdateCheckFirst, sendCommandCheckFirst, kw
+from community.idealarm import custom
```

Comment:
I had something similar while testing out core.utils. I had created the module, saved it, ran a script but it had an error, fixed it but didn't run it and then made changes to core.utils. I thought I had to reload it, but kept getting errors at the reload. Turned out the script had another error in it, so I couldn't reload the module because it had never been loaded. :roll_eyes: Fixed the error in the script and all was well.